File: JavaScript/8js 3/smash.js
/*
|--------------------------------------------------------------------------
| Sentence Smash
|--------------------------------------------------------------------------
|
*/

function smash(words) {
   return words.join(' ');
}


const smash = words => words.join(' ');


function smash(words) {
   let res = '';
   for (let i of words) {
      res += i.concat(' ')
   }
   return res.trim();
}


function smash(words) {
   let res = '';
   let i = 0;
   while (i < words.length) {
      res += words[i].concat(' ')
      i++;
   }
   return res.trim();
}


console.log(smash(["hello"]), "hello");
console.log(smash(["hello", "world"]), "hello world");

File: Python/8py 3/smash.py
"""
|--------------------------------------------------------------------------
| Sentence Smash
|--------------------------------------------------------------------------
|
"""

def smash(words):
   return ' '.join(words)


smash = lambda x:' '.join(x)


import re
def smash(words):
   return re.sub("[]'',[]", "", str(words))


def smash(words):
   str = ""
   for w in words:
      str += w + " "
   return str[:-1]
    

print(smash(["hello"]), "hello")
print(smash(["hello", "world"]), "hello world")